website_sale_infinite_scroll/static/src/js/main.js

```diff
@@ -42,10 +42,17 @@ odoo.define("website_sale_infinite_scroll.main", function (require) {
             if (current_url.indexOf("/shop") !== -1) {
                 var self = this;
                 const container = this.el;
+                let lastScrollTop = 0;
+                const btn_back_to_top = document.querySelector(".btn-back-to-top");
```

Comment:
**nitpick:** please, be consistent with the variable names and always use camel case according to the [JavaScript Coding Conventions](https://www.w3schools.com/js/js_conventions.asp)

Reply:
Thanks a lot for the tips)